components/grid/columns/bound.md

```diff
@@ -1,17 +1,19 @@
 ---
 title: Data Binding
-page_title: Grid - DataBound Column
+page_title: Data Bound Columns | Grid for Blazor
```

Comment:
Shall we consider this a better directive for the page title? A more common used one amongst the documentation is Component - FooBar